src/layouts/Footers/Footer/index.spec.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 import { render } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
 import { BrowserRouter as Router } from 'react-router-dom'
-import '../../i18n'
+import 'src/i18n'
 
```

Comment:
Why sometimes relative pathes and sometimes absolute ones are used?

What's best practics here?